### DISCUSSION TITLE
date setting - help

### DISCUSSION THREAD

**Discussion by Rafalbala** [joshcarty/google-searchconsole#63] [Q&A]
Hi, 
How to modify the script so that I can set the date:  from 1/01/2022 to 1/12/2022

**Comment by joshcarty** [comment]
You can modify this line  from the example in the Quickstart to:

```python
report = webproperty.query.range('2022-01-01', '2022-12-01').dimension('query').get()
```

There are more examples of to use the `range` method [here](https://github.com/joshcarty/google-searchconsole/blob/master/searchconsole/query.py#L85).
